Keep-Me-Alive/src/main/java/com/apprentices/KeepMeAlive/entity/Plant.java

```diff
@@ -0,0 +1,21 @@
+package com.apprentices.KeepMeAlive.entity;
+
+import javax.persistence.*;
+import lombok.Data;
+
+@Data
```

Comment:
Would the @Data allow us to set the id again if its not a final field? Should we make this a final field whilst we are at it?